logistics/scripts/modules/settings/settings-management.psm1

```diff
@@ -337,9 +433,9 @@ function Add-TestHarnessSpecificAppSettings([hashtable] $Settings = @{ }, [strin
 
 function Get-UserSecretsIdByProject {
     return @{
-        "Application/EdFi.Ods.SandboxAdmin" = "f1506d66-289c-44cb-a2e2-80411cc690ea"
-        "Application/EdFi.Ods.SwaggerUI" = "f1506d66-289c-44cb-a2e2-80411cc690eb"
-        "Application/EdFi.Ods.WebApi" = "f1506d66-289c-44cb-a2e2-80411cc690ec"
+        "Application/EdFi.Ods.SandboxAdmin"               = "f1506d66-289c-44cb-a2e2-80411cc690ea"
+        "Application/EdFi.Ods.SwaggerUI"                  = "f1506d66-289c-44cb-a2e2-80411cc690eb"
+        "Application/EdFi.Ods.WebApi"                     = "f1506d66-289c-44cb-a2e2-80411cc690ec"
         "Application/EdFi.Ods.Api.IntegrationTestHarness" = "f1506d66-289c-44cb-a2e2-80411cc690ed"
```

Comment:
This is actually a little concerning, each project should have its own unique id, they shouldn't be sharing a secrets file.

Reply:
I read it the same way the first time, but the last letter of GUID is unique

Reply:
Well look at that... these were obviously hand crafted.